### PR TITLE
Add Zelda: Seasons

### PR DESCRIPTION
**Pull request by isuckhc**
Adds Zelda: Seasons.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/zelda-seasons/game.dat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title = "Zelda: Seasons"
```
Comment (Leo626) [inline games/zelda-seasons/game.dat]: Use the correct title - `The Legend of Zelda: Oracle of Seasons`.

File: games/zelda-seasons/game.dat [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+description = "The Legend of Zelda: Oracle of Seasons is one of two The Legend of Zelda titles released for the Game Boy Color as part of Capcom's Oracle series, the other being Oracle of Ages, both representing the seventh and eighth main installments of the series."
```
Comment (Leo626) [inline games/zelda-seasons/game.dat]: Word it like this so it states the genre and the developer.

> The Legend of Zelda: Oracle of Seasons is an action-adventure game in The Legend of Zelda series, developed by Flagship (a subsidiary of Capcom). It is one of two The Legend of Zelda titles released for the Game Boy Color as part of Capcom's Oracle series, the other being Oracle of Ages, both representing the seventh and eighth main installments of the series.

File: games/zelda-seasons/game.dat [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "vc"
+vc_system = "gbc"
+
+[[releases]]
+title = "0004000000058D00"
+region = "eur"
+release_date = "2001-03-27"
```
Comment (Leo626) [inline games/zelda-seasons/game.dat]: The date should be for the Virtual Console release - `2013-05-30`.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

File: games/zelda-seasons/game.dat (hunk 3 above)
Comment (isuckhc) [reply games/zelda-seasons/game.dat]: I will keep all of this feedback in mind for future contributions.

**Pull request closed without merging**